server/svix-server/src/cfg.rs

```diff
@@ -128,6 +131,14 @@ pub enum QueueType {
     Redis,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum CacheType {
+    Memory,
+    Redis,
+    None,
```

Comment:
Should we print a warning when people choose `None`? Something like "running with caching disabled which will negatively affect performance and cause things like idempotency not to work."?